src/guides/v2.4/graphql/mutations/add-products-to-cart.md

```diff
@@ -0,0 +1,314 @@
+
+**Request:**
+
+```graphql
+mutation {
+  addProductsToCart(
+    cartId: "HbpLADRmSo5h2dCdF85O5wCaVnrworKL"
+    cartItems: [
+      {
+        quantity: 1
+        sku: "WSH12"
+        selected_options: ["Y29uZmlndXJhYmxlLzkzLzExNA==","Y29uZmlndXJhYmxlLzE5OS8yMzI="]
+      }
+    ]
+  ) {
+    cart {
+      items {
+        id
+        product {
+          name
+          sku
+        }
+        ... on ConfigurableCartItem {
+          configurable_options {
+            id
+            option_label
+            value_id
+            value_label
+          }
+        }
+        quantity
+      }
+    }
+  }
+}
+```
+
+**Response:**
+
+```json
+{
+  "data": {
+    "addProductsToCart": {
+      "cart": {
+        "items": [
+          {
+            "id": "348",
+            "product": {
+              "name": "Erika Running Short",
+              "sku": "WSH12"
+            },
+            "configurable_options": [
+              {
+                "id": 93,
+                "option_label": "Color",
+                "value_id": 114,
+                "value_label": "Green"
+              },
+              {
+                "id": 199,
+                "option_label": "Size",
+                "value_id": 232,
+                "value_label": "28"
+              }
+            ],
+            "quantity": 1
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+`entered_options` - option entered by a customer like: text field, image, etc.
+
+Add a product with entered options to a cart.
+
+**Request:**
+
+```graphql
+mutation {
+  addProductsToCart(
+    cartId: "HbpLADRmSo5h2dCdF85O5wCaVnrworKL"
+    cartItems: [
+      {
+        quantity: 1
+        sku: "24-WG03"
+        entered_options: [{
+          uid: "Y3VzdG9tLW9wdGlvbi81Mg=="
+          value: "Test Engraving"
+        }]
+      }
+    ]
+  ) {
+    cart {
+      items {
+        id
+        product {
+          name
+          sku
+        }
+        ... on SimpleCartItem {
+          customizable_options {
+            id
+            label
+            values {
+              id
+              value
+            }
+          }
+        }
+        quantity
+      }
+    }
+  }
+}
+```
+
+**Response:**
+
+```json
+{
+  "data": {
+    "addProductsToCart": {
+      "cart": {
+        "items": [
+          {
+            "id": "350",
+            "product": {
+              "name": "Clamber Watch",
+              "sku": "24-WG03"
+            },
+            "customizable_options": [
+              {
+                "id": 52,
+                "label": "Engraving",
+                "values": [
+                  {
+                    "id": 1184,
+                    "value": "Test Engraving"
+                  }
+                ]
+              }
+            ],
+            "quantity": 1
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+We can consider `selected_option` and `entered_option` as a unique identifier. They meet the criteria:
```

Comment:
I would say UID of `selected_option` and `entered_option`. The option itself is not an identifier